guides/beginner.md

```diff
@@ -7,57 +7,61 @@ nav_order: 1
+Choose a dose that best suits your basket. Underdosing can lead to a soupy puck (though this does not always indicate a bad shot), while overdosing can cause your puck to contact the shower screen, leading to uneven flow of water through the puck.
 
-Also note that higher doses of coffee are harder to properly extract, as there is more coffee to extract fully. This is especially true for lighter roasts. If you find yourself struggling to extract enough, and your shots are tasting sour despite pulling a longer ratio, try dosing 1-2g less.
+Also note that higher doses of coffee are harder to extract properly, as there is more coffee to extract _fully_. This is especially true for lighter roasts. If you find yourself struggling to achieve adequate extraction levels, and your shots are tasting sour despite pulling a longer ratio, try dosing 1-2g less in the basket.
 
 _For the sake of this example, lets say we are locking in a dose of 18g_
 {: .text-center }
 
 Note: While dose is a variable that can also be adjusted to affect shot time, for simplicity's sake, it is easier to lock in dose *first* and adjust shot time via changing grind size.
 
 ### 2. Quick grind size adjustment
-Once dose is locked in, adjust your grind size so that your shot parameters fall in the ballpark of a 1:2 ratio shot taking 25-30s.
-Don't agonize over this too much, we will refine our grind size later.
+Once your dose is locked in, adjust your grind size so that your shot parameters fall within the ballpark of a 1:2 ratio shot taking 25-30s.
+Don't agonize too much over this step, we will refine our grind size later.
 
 ### 3. Adjust brew ratio to taste
-Once your shot is pulling in the ballpark of 20-35s, we can now adjust brew ratio according to what the shot tastes like.
+Once your shot is pulling in the ballpark of 20-35s, we can now adjust brew ratio based to what the shot tastes like.
 
-A 1:2 brew ratio with a 18g dose would give us a yield of 36g espresso out.
+A 1:2 brew ratio with a 18g dose in would give us a yield of 36g espresso out.
 
-Is the shot tasting too sour? Increase your brew ratio and try pulling 18g/40g, or even 18/44g
+Is the shot tasting too sour? Increase your brew ratio and instead try pulling a 18g/40g, or even 18/44g ratio.
 
-Is the shot tasting too bitter? Shorten the ratio and instead try 18g/32g
+Is the shot tasting too bitter? Shorten the ratio and instead try 18g/32g.
 
-If your shot is tasting both bitter and sour at the same time, you’re probably extracting unevenly due to channeling - work on your puck prep!
+If your shot is tasting both bitter and sour at the same time, you’re probably extracting unevenly due to channeling - [work on your puck prep!](https://espressoaf.com/guides/puckprep.html)
```

Comment:
Just using the relative path should be enough, a little more robust in case anything changes
```suggestion
If your shot is tasting both bitter and sour at the same time, you’re probably extracting unevenly due to channeling - [work on your puck prep!](/guides/puckprep)
```